man/get_repos_urls.Rd

```diff
@@ -0,0 +1,39 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/gitstats_functions.R
+\name{get_repos_urls}
+\alias{get_repos_urls}
+\title{List repository URLS}
+\usage{
+get_repos_urls(gitstats_object, with_files = NULL, verbose = TRUE)
```

Comment:
Can we add parameter regarding shape of URL list returned - right now if we set `with_files = NULL` we get API URL vector which may now be informative e.g if we want just check for existing repos. It would be helpful to have option to see Web URLs as well.

Reply:
Applied. Please re-review.